app/controllers/users_controller.rb

```diff
@@ -2,33 +2,28 @@ class UsersController < ApplicationController
   def new; end
 
   def create
-    BackEndFacade.create_user(params)
-    found_user = BackEndFacade.get_user(params[:google_id])
+    UserFacade.create_user(params)
+    found_user = UserFacade.get_user(params[:google_id])
     if found_user.present?
       session[:google_token] = params[:google_token]
       session[:google_id] = params[:google_id]
-      redirect_to dashboard_path(found_user.id)
+      redirect_to dashboard_index_path
     else
       flash[:error] = "Couldn't create your account, please try again."
       redirect_to root_path
     end
   end
 
-  def dashboard
-    @user_friends = BackEndFacade.get_user_friends(current_user.id)
-    @user_gyms = BackEndFacade.get_user_gyms(current_user.id)
-    @user_events = BackEndFacade.get_user_events(current_user.id)
-  end
-
-  def profile
-    current_user_friends = BackEndFacade.get_user_friends(current_user.id)
-    @profile_user = BackEndFacade.get_profile_user(params[:user_id])
-    @user_friends = BackEndFacade.get_user_friends(@profile_user.id)
+  # User Profile
+  def show
```

Comment:
Agree with approach - this makes more sense now

Reply:
Thanks!  There were some endpoints I struggled to decide which on to put them in.  So if you think that can be reorganized better, let me know!